core/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala

```diff
@@ -858,6 +858,21 @@ class SQLPlanParserSuite extends BaseTestSuite {
     }
   }
 
+  test("Python UDFs should be supported") {
+    val eventLog = s"$qualLogDir/python_udf_eventlog"
+    val pluginTypeChecker = new PluginTypeChecker()
+    val app = createAppFromEventlog(eventLog)
+    assert(app.sqlPlans.size == 1)
+    val parsedPlans = app.sqlPlans.map { case (sqlID, plan) =>
+      SQLPlanParser.parseSQLPlan(app.appId, plan, sqlID, "", pluginTypeChecker, app)
+    }
+    val execInfo = getAllExecsFromPlan(parsedPlans.toSeq)
+    val projectExec = execInfo.filter(_.exec.contains("Project"))
+    assertSizeAndSupported(2, projectExec)
+    val batchEvalPythonExec = execInfo.filter(_.exec.contains("BatchEvalPython"))
+    assertSizeAndSupported(1, batchEvalPythonExec)
```

Comment:
there is something I missing here. I cannot find anywhere in the code that we handle `BatchEvalPython` exec.
How come that the result shows it as supported?

Reply:
BatchEvalPython should not be supported. I will fix this.

Reply:
Have we confirmed that for a fact?
I mean did we verify that it is most likely that `BatchEvalPython` is associated with PythonUDF and PythonUDAF? if that's the case then all we need to do is considering that the entire exec is supported with being conservative on the speedup factor.
As far as I can see in the plugin code, the RAPIDS has very limited capability accelerating those expressions anyway. So, the speedup should be near 1.